airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -119,6 +138,29 @@ def _icon_upload(metadata: ConnectorMetadataDefinitionV0, bucket: storage.bucket
         return False, f"No Icon found at {local_icon_path}"
     return upload_file_if_changed(local_icon_path, bucket, latest_icon_path)
 
+def _docs_upload(metadata: ConnectorMetadataDefinitionV0, bucket: storage.bucket.Bucket, metadata_file_path: Path, latest: bool) -> Tuple[bool, str]:
+    connector_type = metadata.data.connectorType
+    doc_file_name = metadata.data.documentationUrl.split('/')[-1]
+    docs_folder_path = metadata_file_path.parents[3] / DOCS_FOLDER_PATH / f"{connector_type}s"
+    
+    local_doc_path = docs_folder_path / f"{doc_file_name}.md"
+    local_inapp_doc_path = docs_folder_path / f"{doc_file_name}.inapp.md"
+
+    remote_doc_path = get_doc_remote_file_path(metadata.data.dockerRepository, "latest" if latest else metadata.data.dockerImageTag, False)
+    remote_inapp_doc_path = get_doc_remote_file_path(metadata.data.dockerRepository, "latest" if latest else metadata.data.dockerImageTag, True)
+    
+    if local_doc_path.exists():
+        doc_uploaded, doc_blob_id = upload_file_if_changed(local_doc_path, bucket, remote_doc_path)
+    else:
+        doc_uploaded, doc_blob_id = False, f"No doc found at {local_doc_path}"
```

Comment:
❓  Do we want to enforce that they have docs?

Reply:
Yeah I think this should be enforced, I will add an exception here

Reply:
Great. Lets run test on a few GA connectors so that were sure we didnt cause a regression :)